styles.css

```diff
@@ -514,4 +525,80 @@ textarea {
   .intro .connect .connect-title p {
     font-size: 16px;
   }
+
+  .card {
+    flex-direction: row;
+    display: flex;
+  }
+
+  .project:nth-child(even) .card {
+    flex-direction: row-reverse;
+  }
+
+  .card .card-image {
+    width: 100%;
+  }
+
+  .card .card-image img {
+    width: 100%;
+  }
+
+  .card .card-body {
+    width: 50%;
+    margin: 3rem 0 0 2rem;
+  }
+
+  .card .card-body .card-title {
+    width: 100%;
+  }
+
+  .card .card-body .card-client-info {
+    margin: 1rem 0;
+  }
+
+  .card-btn {
+    margin: 4rem 0;
+  }
+
+  .about .about-container {
+    display: flex;
+    padding: 10rem 2rem;
+  }
```

Comment:
- [ ] Please try as much as possible to stick to the Figma template design. In this case, try as much as possible to increase the `size` of the about section heading. In addition, kindly ensure to increase the gap between `container` and `about-tech` to match the Figma design. This will make your portfolio professional and user-friendly ☺️. On this section, finally, please make sure that the HTML icon aligns with the texts in the same container.
- [ ] Please make sure the color of `framework` and `skills` is changed to `#344563` to match the Figma design.

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/228821003-68d0ed14-c22f-404d-b7eb-9c25e798b718.png)

Your design 😎
![image](https://user-images.githubusercontent.com/99394374/228821140-3f78f5f1-3616-4247-8566-055539952c3a.png)